### PR TITLE
Add nix experiment docs

### PR DESCRIPTION
**Pull request by ahobson**
## Description

Document how to use nix for milmove_load_testing

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Follow instructions in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,6 +277,29 @@ out the necessary python commands to run to deactivate your env and remove the c
 
 ```shell script
 make teardown
+### Setup: Nix
```
Comment (gidjin) [inline README.md:280]: Looks like the trailing three ` that should go after `make teardown` are missing which is messing with the format.

File: README.md [hunk 2]
```diff
@@ -274,6 +277,29 @@ out the necessary python commands to run to deactivate your env and remove the c
...
+
+1. First read the overview in the [Truss Engineering Playbook](https://github.com/trussworks/Engineering-Playbook/tree/main/developing/nix).
+1. Follow the [macOS installation instructions](https://nixos.org/manual/nix/stable/#sect-macos-installation).
+1. Ensure you have `direnv` and a modern `bash` installed. To install
+   globally with nix, run `nix-env -i direnv bash`
+1. Ensure you have run `direnv allow` to set up the appropriate nix
+   environment variables.
+1. Make sure you have disabled any `nodeenv`, `asdf` or any other
+   version switchers.
+1. Run `./nix/update.sh`
```
Comment (gidjin) [inline README.md:297]: What should I do after running this command? I ran it but don't have `pytest` which doesn't appear in the `default.nix` file. I tried `make venv` and `make setup` but both failed due to missing `pyenv`. Would be nice to include the rest of the steps to get the tests running if possible.

**Review by gidjin — COMMENTED** [review]
Nix install went fine, but I'm a little lost as to how to get the project going after python is installed by nix. Also I found a formatting issue in the README

File: README.md (hunk 1 above)
Comment (ahobson) [reply README.md:280]: Thanks! I think I fixed that

File: README.md (hunk 2 above)
Comment (ahobson) [reply README.md:297]: I added docs for installing the pip dependencies

**Review by gidjin — APPROVED** [review]
Changes look good to me! 🚀  Please run the `gh-md-toc` tool to update the README toc before merging.

**Comment by ahobson** [comment]
> Changes look good to me! 🚀 Please run the `gh-md-toc` tool to update the README toc before merging.

I can do that, but GitHub now supports it (in a different way)

https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

**Pull request merged**